src/grpc_interop.cc

```diff
@@ -193,6 +210,82 @@ int32_t ServerCleanupProc(grpc_labview::gRPCid* serverId)
+//---------------------------------------------------------------------
+LIBRARY_EXPORT int32_t RegisterReflectionProtoString(grpc_labview::gRPCid** id, grpc_labview::LStrHandle serializedFileDescriptor)
+{
+    std::string serializedDescriptorStr = grpc_labview::GetLVString(serializedFileDescriptor); 
+    
+    auto server = (*id)->CastTo<grpc_labview::LabVIEWgRPCServer>();
+    if (server == nullptr)
+    {
+        return -1;
+    }
+    
+    if (!server->RegisterReflectionProtoString(serializedDescriptorStr)) {
+        return -(1000 + grpc::StatusCode::INVALID_ARGUMENT);
+    }
+
+    return 0;
+}
+
+//---------------------------------------------------------------------
+// DeserializeReflectionInfo is kept only for backwards-compatibility.
+// For LV servers generated using grpc-labview v1.0.0.6 through v1.5.1.1, this DLL function
+// was directly called from the server template (instead of a grpc-labview support
+// wrapper function).  As a result, any LV servers generated using these versions
+// require this exported function.
+// 
+// Because this function does not take a gRPCid input, an array of server instances is kept
+// and iterated through to register all serialized descriptors to all servers as a workaround.  
+// While this may not be correct (as not all servers may use all services), this is the best
+// method of backwards-compatibilty as possible without breaking the previously-generated
+// server code.  
+// 
+// This function will only register the reflection strings to servers which
+// are created _but not running_.  This is to aid in backwards-compatibility.  If multiple
+// servers are being created, it is likely that they are created and started at different times.
+// As a result, only registering services to the server which is not running will likely
+// be the appropriate server.  As a result, all descriptor strings must be passed prior to calling
+// LVStartServer when using this deprecated function.
+// 
+// Starting with grpc-labview v1.6.0.1, only the RegisterReflectionProtoString function
+// above is used, which registers descriptor strings to a specified server instance.
+//---------------------------------------------------------------------
+LIBRARY_EXPORT void DeserializeReflectionInfo(grpc_labview::LStrHandle serializedFileDescriptor)
+{
+    std::string serializedDescriptorStr = grpc_labview::GetLVString(serializedFileDescriptor);
+
+    // For every server instance, check to see if the server is not running by seeing
+    // if the listening port is still 0 (when a server is running listening port != 0)
+    // If the server is not running, register the reflection strings to that server
+    for (auto id : server_instances) {
+        if (id != nullptr) {
+            auto server = (*id).CastTo<grpc_labview::LabVIEWgRPCServer>();
+            if (server == nullptr)
+            {
+                return;
+            }
+            else {
+                if (server->ListeningPort() == 0) {
```

Comment:
nit: Make this an `else if` on a single line to avoid some nesting.

Also, the predominant styling used throughout the repo is for curly braces to be stacked on their own individual lines rather than staggering the opening curly brace at the end of the flow control statement and the closing curly brace on its own line. I know its not everybody's preferred style, but I would prefer to maintain consistency.